recipes/master.rb

```diff
@@ -67,16 +67,25 @@
   end
 end
 
+jenkins_command 'safe-restart' do
+  action :nothing
+end
+
 {
   'credentials' => '2.1.11',
   'credentials-binding' => '1.10'
 }.each do |p, v|
   jenkins_plugin p do
     version v
-    notifies :restart, 'service[jenkins]'
+    notifies :execute, 'jenkins_command[safe-restart]'
   end
 end
 
+jenkins_plugin 'ssh-credentials' do
+  version '1.12'
+  notifies :execute, 'jenkins_command[safe-restart]', :immediately
```

Comment:
Because we need this installed before we can use ``jenkins_private_key_credentials`` further in the cookbook.